2021-10-11 17:37:13,932 INFO    MainThread:17104 [wandb_setup.py:_flush():69] setting env: {}
2021-10-11 17:37:13,932 INFO    MainThread:17104 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-11 17:37:13,932 INFO    MainThread:17104 [wandb_init.py:_log_setup():342] Logging user logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211011_173713-219u1rnf\logs\debug.log
2021-10-11 17:37:13,933 INFO    MainThread:17104 [wandb_init.py:_log_setup():343] Logging internal logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211011_173713-219u1rnf\logs\debug-internal.log
2021-10-11 17:37:13,933 INFO    MainThread:17104 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001998CB01D00>
2021-10-11 17:37:13,934 INFO    MainThread:17104 [wandb_init.py:init():375] calling init triggers
2021-10-11 17:37:13,934 INFO    MainThread:17104 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-11 17:37:13,935 INFO    MainThread:17104 [wandb_init.py:init():424] starting backend
2021-10-11 17:37:13,935 INFO    MainThread:17104 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-10-11 17:37:13,942 INFO    MainThread:17104 [backend.py:ensure_launched():135] starting backend process...
2021-10-11 17:37:14,034 INFO    MainThread:17104 [backend.py:ensure_launched():139] started backend process with pid: 8168
2021-10-11 17:37:14,035 INFO    MainThread:17104 [wandb_init.py:init():429] backend started and connected
2021-10-11 17:37:14,094 INFO    MainThread:17104 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-11 17:37:14,096 INFO    MainThread:17104 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-11 17:37:14,096 INFO    MainThread:17104 [wandb_init.py:init():477] updated telemetry
2021-10-11 17:37:14,097 INFO    MainThread:17104 [wandb_init.py:init():500] communicating current version
2021-10-11 17:37:15,275 INFO    MainThread:17104 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-11 17:37:15,276 INFO    MainThread:17104 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-11 17:37:15,660 INFO    MainThread:17104 [wandb_init.py:init():540] starting run threads in backend
2021-10-11 17:37:15,766 INFO    MainThread:17104 [wandb_run.py:_console_start():1601] atexit reg
2021-10-11 17:37:15,767 INFO    MainThread:17104 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-11 17:37:15,767 INFO    MainThread:17104 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-11 17:37:15,769 INFO    MainThread:17104 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-11 17:37:15,769 INFO    MainThread:17104 [wandb_init.py:init():565] run started, returning control to user process
2021-10-11 17:37:15,773 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:15,773 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:15,873 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:15,874 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:15,875 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:15,930 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:16,014 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:16,014 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:16,077 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:16,078 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:16,078 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:16,130 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:16,131 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:16,131 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:22,905 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:22,907 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:22,908 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:22,949 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:22,949 INFO    MainThread:17104 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-11 17:37:22,950 INFO    MainThread:17104 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-10-11 17:37:23,043 INFO    MainThread:17104 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-11 17:37:23,055 INFO    MainThread:17104 [wandb_setup.py:_flush():69] setting env: {}
2021-10-11 17:37:23,055 INFO    MainThread:17104 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-11 17:37:23,056 INFO    MainThread:17104 [wandb_init.py:_log_setup():342] Logging user logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211011_173723-1dyekvnq\logs\debug.log
2021-10-11 17:37:23,056 INFO    MainThread:17104 [wandb_init.py:_log_setup():343] Logging internal logs to c:\Users\reese\OneDrive\Documents\University\EEE4022S\Workspace\Final-Year-Project\wandb\run-20211011_173723-1dyekvnq\logs\debug-internal.log
2021-10-11 17:37:23,057 INFO    MainThread:17104 [wandb_init.py:init():375] calling init triggers
2021-10-11 17:37:23,057 INFO    MainThread:17104 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-11 17:37:23,057 INFO    MainThread:17104 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 219u1rnf
2021-10-11 17:37:23,058 INFO    MainThread:17104 [wandb_run.py:finish():1271] finishing run rees123/Resnet50 Regression/219u1rnf
